### DISCUSSION TITLE
Bernd Storath (@kaaax0815) joined the wg-easy team as Maintainer

### DISCUSSION THREAD

**Discussion by pheiduck** [wg-easy/wg-easy#1313] [Announcements]
Hello Community,

I am glade to announce that @kaaax0815 joined the team as Maintainer.

He has Contributed many times to this Project. @WeeJeWel and I think that it is just a win to have him on board as he has also Development Experince in TypeScript, Node.js, Express.js and GitHub to name a few.

Please give @kaaax0815 a warm welcome!

**Comment by Decibel0377** [comment]
Welcome

**Reply by kaaax0815** [reply]
👋
